### ISSUE TITLE
ban alert functoin is not work

### ISSUE THREAD

**Issue by giscafer** [palantir/tslint#2913]
### Bug Report

- __TSLint version__: 5.3.2
- __TypeScript version__: 2.3.3
- __Running TSLint via__: node v8,npm v5

#### TypeScript code being linted

```ts
import {Component} from '@angular/core';

@Component({
    selector: 'app-root',
    templateUrl: './app.component.html',
    styleUrls: ['./app.component.scss']
})
export class AppComponent {
    title = 'app';

    constructor() {
        
        alert('sss');//npm lint is not work

    }
}

```

with `tslint.json` configuration:

```json
"ban": [
      true,
      "alert",
      [
        "window",
        "alert",
        "eval"
      ]
    ],
```

#### Actual behavior

`All files pass linting.`

#### Expected behavior

`ERROR: G:/GitHub/111start/ng-starter/src/app/app.component.ts[13, 9]: Calls to 'alert' are not allowed. `

ps:

window.alert is banned，but only alert function is not work.

**Comment by ajafff** [comment]
Use `["alert"]` or upgrade to tslint 5.4.0 which introduced the new option format that accepts plain strings.

**Comment by giscafer** [comment]
@ajafff thks。it work after upgrade.

by the way,Is there any way to lint Function name must be named in camelCase rule?

such as `   "directive-selector": [true, "attribute", "ib", "camelCase"],`

**Comment by ajafff** [comment]
@giscafer Currently there's not rule to lint function names in tslint core.
But there are several third party rules that can do that:

You can use `naming-convention` from https://github.com/ajafff/tslint-consistent-codestyle#naming-convention with this config:

```js
{
    "rulesDirectory": ["tslint-consistent-codestyle"],
    ... // other settings like "extends" that you currently have in your tslint.json
    "rules": {
        ... // other rules that you currently have in your tslint.json
        "naming-convention": [
            true,
            {"type": "function", "format": "camelCase"}
        ]
    }
}
```

Or there's `ext-variable-name` from https://www.npmjs.com/package/vrsource-tslint-rules
